packages/unity-react-core/src/core/utils/index.js

```diff
@@ -130,3 +130,9 @@ export {
   initImageCarousel,
   initImageGalleryCarousel,
 };
+import { SidebarMenu } from "../../components/SidebarMenu/SidebarMenu.tsx";
```

Comment:
Can this import please be put at the top of the file in alpha order with the other imports?